src/FlowtypeRunner.js

```diff
@@ -0,0 +1,48 @@
+const { exec } = require('child_process');
+const path = require('path');
+const { pass, fail } = require('create-jest-runner');
+
+class FlowtypeRunner {
+  constructor(globalConfig) {
+    this.globalConfig = globalConfig;
+  }
+
+  runTests(tests, watcher, onStart, onResult, onFailure, options) {
+    this.globalConfig = {};
+    const start = +new Date();
+    return new Promise((resolve) => {
+      exec('flow', { stdio: 'ignore', cwd: process.cwd() }, (err, stdout) => {
+        const errors = stdout.split('Error');
+        const errorsPerFile = errors.reduce((previous, current) => {
+          const firstErrorLine = current.split('\n')[0];
+          let fileName = firstErrorLine.split(':')[1];
```

Comment:
An unexpected colon before or after the fileName might break this. Might want to use regex.

Reply:
👍 Did it